src/sentry.js

```diff
@@ -0,0 +1,16 @@
+import { init as initSentry } from '@sentry/browser'
+import env from './environment'
+import { getNetworkName } from './lib/web3-utils'
+
+const SENTRY_DSN = env('SENTRY_DSN')
+export const sentryEnabled = Boolean(SENTRY_DSN && env('ENABLE_SENTRY'))
```

Comment:
Should we rely on the presence of `SENTRY_DSN`? Or do you see a reason to have `ENABLE_SENTRY` as well?

Reply:
That's a good point, It was to prevent sending events during local development but in that instance the `SENTRY_DSN` isn't available anyway.